text/0000-manageable-http-proxies.md

```diff
@@ -0,0 +1,65 @@
+- Feature Name: Manageable HTTP Proxies
+- Start Date: 2016-10-03
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+[summary]:
+
+Adds HTTP Proxies as a first class citizen that is manageable by users for use with syncing and potentially other things.
+
+# Motivation
+[motivation]:
+
+Currently when synchronizing repositories, users can either sync straight to the content source, or globally use an http proxy defined at installation time.  In many cases, you may want to use one proxy for anything syncing outside your network, but no proxy for internal sources.  There may be other cases where you want to use one proxy for syncing one repo and another proxy for a second repo.
```

Comment:
Yes, the url for the proxy can be http:// or https:// just like today.  Auth is included, see the below where i talk about the attribute for the proxy (url, username, password)